### PR TITLE
Add proposed dates for the rest of the year to the top-level README

### PR DESCRIPTION
**Pull request by tabatkins**

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]
This is in an autogenerated section of the readme (`npm run build`); it also would create a secondary source of truth that needs updating, so i'm not sure it's the best place to put it.

**Comment by ljharb** [comment]
Although maybe we should move it to the readme _above_ the autogenerated section, and remove it from the agendas themselves (and replace them with a link to the readme section)?

**Comment by tabatkins** [comment]
Ah, had no idea it was auto-genned, but now that comment in the source makes sense. I've moved the table to its own details above the auto-gen section for now; we can think about exposing this information to the autogen later.

**Comment by ljharb** [comment]
Sounds great! Can you also remove the table from the template and the January agenda, and make both link to the new table in the readme?

**Comment by michaelficarra** [comment]
Also, the January meeting is still upcoming!

**Comment by tabatkins** [comment]
Jan meeting is upcoming, but confirmed and with its own page already, so it doesn't go in the "proposed" table. ^_^

**Comment by jridgewell** [comment]
> Jan meeting is upcoming, but confirmed and with its own page already, so it doesn't go in the "proposed" table. ^_^

I would make this a "future meetings" section instead of a "**tentatively planned** future meetings", with a "confirmed" field on the table

**Comment by tabatkins** [comment]
I added a "Confirmed" column, but since I waited so long, the Jan meeting is no longer upcoming at all. ^_^

File: README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@ agendas
 =======
 
 TC39 meeting agendas
+
+<details>
+<summary>Future Meetings</summary>
+
+| Confirmed | Future Dates             | Location            | Host                 |
+|-----------|--------------------------|---------------------|----------------------|
+|           | 2023-03-21 to 2023-03-23 | Seattle, US         | F5                   |
```
Comment (ljharb) [inline README.md:11]: i think this one's pretty confirmed

**Comment by tabatkins** [comment]
Cool, put a checkmark in its row now

**Review by jridgewell — APPROVED** [review]

**Pull request merged**